AppInspector.CLI/CLICmdOptions.cs

```diff
@@ -19,6 +19,37 @@ public class CLICommandOptions : CommandOptions
         public string OutputFileFormat { get; set; } = "text";
     }
 
+    /// <summary>
+    /// CLI command distinct arguments
+    /// </summary>
+    [Verb("get-tags", HelpText = "Inspect source directory/file/compressed file (.tgz|zip) against defined characteristics for unique tags")]
+    public class CLIGetTagsCommandOptions : CLICommandOptions
+    {
+        [Option('s', "source-path", Required = true, HelpText = "Source file or directory to inspect")]
+        public string? SourcePath { get; set; }
+
+        [Option('r', "custom-rules-path", Required = false, HelpText = "Custom rules file or directory path")]
+        public string? CustomRulesPath { get; set; }
+
+        [Option('i', "ignore-default-rules", Required = false, HelpText = "Exclude default rules bundled with application", Default = false)]
+        public bool IgnoreDefaultRules { get; set; }
+
+        [Option('c', "confidence-filters", Required = false, HelpText = "Output only matches with specified confidence <value>,<value> [high|medium|low]", Default = "high,medium")]
+        public string ConfidenceFilters { get; set; } = "high,medium";
+
+        [Option('k', "file-path-exclusions", Required = false, HelpText = "Exclude source files (none|default: sample,example,test,docs,lib,.vs,.git)", Default = "sample,example,test,docs,lib,.vs,.git")]
+        public string FilePathExclusions { get; set; } = "sample,example,test,docs,.vs,.git";
+
+        [Option("file-timeout", Required = false, HelpText = "Maximum amount of time in seconds to allow for processing each file.", Default = 0)]
+        public int FileTimeOut { get; set; } = 0;
+        
+        [Option("single-threaded", Required = false, HelpText = "Disables parallel processing.")]
```

Comment:
At some point, we should probably remove this option, I think?

Reply:
It's useful for low power systems and very useful for debugging.